api/ReadableStream.json

```
@@ -418,6 +418,72 @@
             "deprecated": false
           }
         }
+      },
+      "values": {
+        "__compat": {
+          "spec_url": "https://streams.spec.whatwg.org/#readablestream",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "110"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "@@asyncIterator": {
+        "__compat": {
+          "spec_url": "https://streams.spec.whatwg.org/#readablestream",
+          "support": {
```

It should be either this OR the `async_iterable` feature above, but not both

My objection to this one is that the symbol is not "well known" or understood - i.e. no one will look at this and know what it is. How about we update this so that it is more useful

```suggestion
      "@@asyncIterator": {
        "__compat": {
          "description": "Async iterable (supports iteration using <a href="https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/for-await...of"><code>for await ... of</code></a>)",
          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Symbol/asyncIterator",        
          "spec_url": "https://streams.spec.whatwg.org/#rs-asynciterator",
          "support": {
```

Then delete the async_iterable key.